Stp.Data/Entities/TaskAndSkill.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace Stp.Data.Entities
+{
+    [Table("TaskAndSkill")]
+    public class TaskAndSkill
+    {
+        public long Id { get; set; }
+        [ForeignKey(nameof(StpTask))]
+        public long TaskId { get; set; }
+        [ForeignKey(nameof(StpSkill))]
+        public long SkillId { get; set; }
```

Comment:
In ForeignKey attribute you need to use Property name, not Type name. So it should be Task and Skill (not StpTask and StpSkill)